DOCS/man/input.rst

```diff
@@ -2139,6 +2139,18 @@ Property list
     ``playback-time/full``
         ``playback-time`` with milliseconds.
 
+``remaining-file-loops``
```

Comment:
wouldn't `loop-file-remaining` be more in line with existing naming?

Reply:
That makes no grammatical sense though. And the options are already inconsistent, `--loop-file-count` and `--loop-playlist-count` would more logical and consistent with `--ab-loop-count`.